### PR TITLE
Upload tasks: PseudoRandomStream, Integrate, FunctionalInterface

### PR DESCRIPTION
**Pull request by ymilenin**

### CODE REVIEW

File: src/com/lesson12/second/IntegrateApp.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import java.util.function.DoubleUnaryOperator;
+
+public class IntegrateApp {
+    public static void main(String[] args) {
+        System.out.println(integrate(x -> 1, 0, 10));
+        System.out.println(integrate(x -> 1, 8, 20));
+    }
+
+    private static double integrate(DoubleUnaryOperator f, double a, double b) {
+        int n = 1000000;
```
Comment (Sofasmile) [inline src/com/lesson12/second/IntegrateApp.java]: better:
```suggestion
        int n = 1_000_000;
```

File: src/com/lesson12/second/IntegrateApp.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        double result = 0;
+        double gridStep = (b - a) / n;
+
+        for(int i = 0; i < n; i++) {
+            result += gridStep * f.applyAsDouble(a);
+        }
+
```
Comment (Sofasmile) [inline src/com/lesson12/second/IntegrateApp.java]: remove redundant line

File: src/com/lesson12/second/IntegrateApp.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import java.util.function.DoubleUnaryOperator;
+
+public class IntegrateApp {
+    public static void main(String[] args) {
+        System.out.println(integrate(x -> 1, 0, 10));
+        System.out.println(integrate(x -> 1, 8, 20));
+    }
+
+    private static double integrate(DoubleUnaryOperator f, double a, double b) {
```
Comment (Sofasmile) [inline src/com/lesson12/second/IntegrateApp.java]: does this method work on stepik?

Comment (ymilenin) [reply src/com/lesson12/second/IntegrateApp.java]: Sorry, upload the old version of the program)
New version works on Stepik
image

**Pull request merged**